### PR TITLE
 Add openjdk8 to travis (to ensure portability) 

### PR DESCRIPTION
**Pull request by treba123**
see https://github.com/loklak/loklak_server/issues/783 https://github.com/loklak/loklak_server/issues/782

Edit2: reverted the start script changes. they will come together with the installation page.

Edit: made some adjustments on the start script. It should now fail correctly if loklak can't be launched at all.
Before merging please test what happens when you do
- ant clean
- bin/start.sh

### CODE REVIEW

**Comment by DengYiping** [comment]
@treba123 Please squash commit. Wait for #788 to be merged, so that you can pass travis-ci test.Cheers!

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 language: java
+sudo: required
```
Comment (sudheesh001) [inline .travis.yml]: Is there a reason why we need sudo or is it just to make the builds faster ?

**Comment by treba123** [comment]
Without sudo, we can't use trusty for some reason. Tried it without, didn't work.

File: .travis.yml (hunk 1 above)
Comment (treba123) [inline .travis.yml]: Yes, trusty only works with it. And we need trusty for openjdk8

**Pull request merged**